src/modules/esl-event-listener/core/targets/resize.adapter.ts

```diff
@@ -10,6 +15,14 @@ export class ESLResizeObserverTarget extends SyntheticEventTarget {
     changes.forEach(this.handleChange, this)
   );
 
+  public readonly target: Element;
+
+  private static getActualTarget(target: ESLResizeTarget): Element {
+    if (target instanceof Element) return target;
+    const host = getPropertyDescriptor(target, '$host') as Element;
```

Comment:
How does the property descriptor relate to the host value ? 
Is it not compatible;e with Element, and it is not something that the ESLResizeObserverTarget observer should use.